sopel/modules/tld.py

```diff
@@ -202,8 +211,12 @@ def get_processed_data(self):
                     tld_list[idn_key] = zipped
 
         LOGGER.debug("Finished processing TLD data; returning it.")
-        self.tables = tld_list
         self.finished = True
+        # clear working data
```

Comment:
Should I amend this to say why (to let the temporary data be garbage collected rather than holding a ref to it)?

Reply:
A why is always welcome, yet not strictly necessary here.